doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+
+!media neams_workbench/neams_workbench_ncrc_if.png style=width:80% id=fig:neams_workbench_ncrc_if caption=Input file (left) and mesh visualization (right) with the NEAMS Workbench.
+
+### Step 3: running a job from the NEAMS Workbench GUI
+
+After editing the input file and making sure that the physics enabled are consistent with the geometry visualized with ParaView, the input file is submitted to the scheduler. The NEAMS Workbench has already been set to run this input file with the _BlueCRAB_ executable (configured in Step 1). To submit the job to `Sawtooth`, simply type `Run`, which will create the required job scheduler script behind the scenes and put your job in the queue. To customize your job run options first, click the downward arrow next to `Run` and select `Customize Run Options...`. A widget opens that allows users to modify the number of cores and CPU time by scrolling down to `Remote: Scheduler Header` and modifying the number of nodes (select), numbers of CPUs per node (ncpus) and walltime (walltime), as illustrated in [fig:hpc_on_demand_pbs_options]. The suggested number of nodes/CPUs and walltime should be given in the VTB documentation for each test case. Progress of the job is displayed in the `Message`: the NEAMS Workbench checks on the status of the job and tail the output once it starts to run on Sawtooth. Once the job completes, it will display `Processed Finished`.
+
+!media neams_workbench/hpc_on_demand_pbs_options.png style=width:80% id=fig:hpc_on_demand_pbs_options caption=Editing PBS options with the NEAMS Workbench GUI.
+
+### Step 4: visualize the numerical solution from the NEAMS Workbench GUI
+
+Visualization of the numerical solution is also achieved within the NEAMS Workbench GUI. Users can refer to the steps below or watch the *[demonstration video](https://www.youtube.com/watch?v=W6Rq_owHmoU&ab_channel=MarcoDelchini)* starting at 4:20. As for visualizing the geometry and the mesh described in Step 1, the Exodus file that contains the numerical solution can be visualized with the built-in ParaView, as shown in [fig:neams_workbench_sfr_vis]. The same steps as in 4.2.1 can be followed. An alternative way is to right click on the name of the input file in the `Navigation` tab and select `Open working directory`. From there, a navigation window opens with all files contained in the working directory. The Exodus file `griffin_out.e` can be dragged and dropped in the NEAMS Workbench GUI. A *visualization tab* or *render view* should automatically open and display the geometry as shown in [fig:hpc_on_demand_render_view].
```

Comment:
you could make the youtube links point to the right second directly

Reply:
What do you mean by `the right second directly`?

Reply:
as in the 'second' (second as in second/minute/hour) in the video that corresponds to what you are saying

Reply:
done